tests/test_curves.py

```diff
@@ -55,19 +55,15 @@ def test_torsion(self):
     def test_frenet(self):
         """Test frenet-serret frame of circular curve."""
         c = FourierRZCurve()
-        data = c.compute(
-            ["frenet_tangent", "frenet_normal", "frenet_binormal"], basis="rpz", grid=0
-        )
+        data = c.compute(["frenet_tangent", "frenet_normal", "frenet_binormal"], grid=0)
         T, N, B = data["frenet_tangent"], data["frenet_normal"], data["frenet_binormal"]
         np.testing.assert_allclose(T, np.array([[0, 1, 0]]), atol=1e-12)
         np.testing.assert_allclose(N, np.array([[-1, 0, 0]]), atol=1e-12)
         np.testing.assert_allclose(B, np.array([[0, 0, 1]]), atol=1e-12)
         c.rotate(angle=np.pi)
         c.flip([0, 1, 0])
         c.translate([1, 1, 1])
-        data = c.compute(
-            ["frenet_tangent", "frenet_normal", "frenet_binormal"], basis="xyz", grid=0
-        )
+        data = c.compute(["frenet_tangent", "frenet_normal", "frenet_binormal"], grid=0)
```

Comment:
same as above comment

Reply:
@ddudt

Reply:
@dpanici I reverted this change, but are you sure this test is supposed to use both bases? It's difficult for me to visualize the geometry that is being tested here, since there are flips/rotations/etc. Same for the other two Frenet tests in this file.

Reply:
FYI with this reversion the test is now failing, but I think that's an unrelated bug that I still need to track down.

Reply:
Actually this change is needed to make the test pass (both compute calls need to return "rpz" basis). I'm about 80% sure the old test was incorrect and it is fixed now, but whoever wrote this original test should check it.

Reply:
yes they should both be in rpz. It's just testing the directions of the frenet frame before and after flip/rotate etc. It's not messing with any basis stuff.

Reply:
@f0uriest there are 3 different `test_frenet` functions in this file for different curve parameterizations:

1. `FourierRZCurve` now uses `basis="rpz"` for both computes
2. `FourierXYZCurve` uses `basis="rpz"` for the first compute and `basis="xyz"` for the second compute
3. `FourierPlanarCurve` uses `basis="xyz" for both computes

All three tests now pass, but should they really all be using "rpz" throughout?

Reply:
Iirc when I made them I was using whatever the "native" basis was for the curve, but I don't know how much has changed since then